code/modules/mob/living/simple_animal/hostile/cockroach.dm

```diff
@@ -45,6 +49,43 @@
 /mob/living/simple_animal/hostile/cockroach/add_cell_sample()
 	AddElement(/datum/element/swabable, CELL_LINE_TABLE_COCKROACH, CELL_VIRUS_TABLE_GENERIC_MOB, 1, 7)
 
+/mob/living/simple_animal/hostile/cockroach/proc/splat() //Forces a cockroach death, used for retribution
```

Comment:
This function looks pointless, callnig `death()` looks equivalent. What is it for?